addons/purchase_stock/models/purchase.py

```diff
@@ -344,6 +344,10 @@ def write(self, values):
             new_date = fields.Datetime.to_datetime(values['date_planned'])
             self.filtered(lambda l: not l.display_type)._update_move_date_deadline(new_date)
         lines = self.filtered(lambda l: l.order_id.state == 'purchase')
+
+        if 'product_packaging_id' in values:
+            self.move_ids.group_id.stock_move_ids.product_packaging_id = values['product_packaging_id']
```

Comment:
check if move_orig_ids have the same product_packaging_id than current move. If yes propagate its change to its move_orig_ids

Reply:
with a state not in cancel or done